client/src/pages/Profile.tsx

```
@@ -316,261 +278,106 @@ const Dashboard: React.FC<DashboardProps> = ({ loggedInUsername }) => {
+                </CardContent>
+              </Card>
+            </div>
+            <div className="space-y-4">
+              <Card>
+                <CardHeader>
+                  <h2 className="text-lg font-bold">GitHub Streak</h2>
+                </CardHeader>
+                <CardContent>
+                  {githubStreakSvg ? (
+                    <div
+                      dangerouslySetInnerHTML={{ __html: githubStreakSvg }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)